documentation/release_notes.rst

```diff
@@ -8,6 +8,60 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.4
+=============
+
+New Features
+------------
+-  Added the range-based versions of the following algorithms: ``any_of``, ``adjacent_find``,
+   ``copy_if``,  ``remove_copy_if``, ``remove_copy``, ``none_of``, ``swap_ranges``, ``unique``, ``unique_copy``,
+   ``reverse``, ``reverse_copy``, ``rotate_copy``, ``replace_copy``,  ``replace_copy_if``.
+-  Added non-blocking versions of the following algorithms: ``inclusive_scan``, ``exclusive_scan``,
+   ``transform_inclusive_scan``, ``transform_exclusive_scan``.
+-  Added structured binding support for ``zip_iterator::value_type``.
+
+Changes to Existing Features
+----------------------------
+-  Reworked several internal hetero pattern (reduce,scan) to enable async behavior.
```

Comment:
I do not feel like this is important to know for developers. If important, it needs to be improved to show the impact on developers.

Reply:
@reble, what do you think?